Content.Server/Administration/Commands/BriefCommand.cs

```diff
@@ -0,0 +1,150 @@
+                {
+                    if (args[3].ToLower() == "true")
+                    {
+                        didYouBrief = true;
+                    }
+                }
+
+                foreach (var officer in _entities.EntityQuery<BriefOfficerComponent>(true))
+                {
+                    if (officer.Owner == mind.VisitingEntity)
+                    {
+                        didYouBrief = true;
+                        player.ContentData()!.Mind?.UnVisit();
+                        _entities.QueueDeleteEntity(officer.Owner);
+                        return;
+                    }
+                }
+
+                if (didYouBrief == false)
+                {
+                    shell.WriteError("You are visiting something other than a brief already.");
+                    return;
+                }
+            }
+
+            var outfit = "CentcomGear";
+            if (args.Length >= 1)
+            {
+                if (_prototypeManager.TryIndex<StartingGearPrototype>(args[0], out var outfitProto))
+                {
+                    outfit = outfitProto.ID;
+                }
+                else
+                {
+                    shell.WriteError("Given outfit is invalid.");
+                    return;
+                }
+            }
+
+            var coordinates = player.AttachedEntity != null
+                ? _entities.GetComponent<TransformComponent>(player.AttachedEntity.Value).Coordinates
+                : _entitysys.GetEntitySystem<GameTicker>().GetObserverSpawnPoint();
+
+            var entName = "MobHuman";
+            if (args.Length >= 3)
+                entName = args[2];
+            _prototypeManager.TryIndex<EntityPrototype>(entName, out var entProto);
+            if (entProto == null)
+            {
+                shell.WriteError("Entity prototype is invalid.");
+                return;
+            }
+
+            var brief = _entities.SpawnEntity(entName, coordinates);
+            _entities.EnsureComponent<BriefOfficerComponent>(brief);
+            _entities.TryGetComponent<TransformComponent>(brief, out var briefTransform);
+            if (briefTransform != null)
+                briefTransform.AttachToGridOrMap();
+
+            if (args.Length >= 2)
+                _entities.GetComponent<MetaDataComponent>(brief).EntityName = args[1];
+
+            mind.Visit(brief);
+            SetOutfitCommand.SetOutfit(brief, outfit, _entities);
+        }
+
+        public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
+        {
+            if (args.Length == 1)
+            {
+                var options = IoCManager.Resolve<IPrototypeManager>()
+                    .EnumeratePrototypes<StartingGearPrototype>()
+                    .Select(p => new CompletionOption(p.ID))
+                    .OrderBy(p => p.Value);
+
+                return CompletionResult.FromHintOptions(options, Loc.GetString("brief-command-arg-outfit"));
```

Comment:
```suggestion
                return CompletionResult.FromHintOptions(CompletionHelper.PrototypeIDs<StartingGearPrototype>(), Loc.GetString("brief-command-arg-outfit"));
```

same with all the other usages of this